azure-pipelines.yml

```diff
@@ -2,7 +2,14 @@ name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
 variables:
   CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
   CIBW_MANYLINUX_I686_IMAGE: manylinux2014
-  JSOC_EMAIL: "jsoc@cadair.com"  # For the example gallery on the dev deps build
+
+schedules:
```

Comment:
I don't think we need a cron build at the moment.

Reply:
I'm of the view that there's not point having runs unless we need them (there's a cost to azure/microsoft, and a carbon footprint impact too)

Reply:
But it's not really a cost we have to worry.